py/gc.c

```diff
@@ -298,7 +298,8 @@ STATIC void gc_sweep(void) {
                 #if MICROPY_PY_GC_COLLECT_RETVAL
                 MP_STATE_MEM(gc_collected)++;
                 #endif
-            // fall through to free the head
+                // fall through to free the head
+                MP_FALLTHROUGH
```

Comment:
A nice bonus: uncrustify now indents the comment to the correct level because it sees `MP_FALLTHROUGH` as a statement within this case.